templates/plugin/salt_master.conf.erb

```diff
@@ -39,6 +44,10 @@ rest_cherrypy:
   ssl_key: <%= @foreman_ssl_key %>
   ssl_crt: <%= @foreman_ssl_cert %>
 
+netapi_enable_clients:
+  - local
+  - local_async
```

Comment:
Should this be configurable?

Reply:
Good catch, I'll make it configurable.

Do you think tftp:packages ([here](https://github.com/theforeman/puppet-foreman_proxy/blob/32de4bd93998a7dad6a05f20f05f18536ba100a2/manifests/tftp/netboot.pp#L4)) is a good example which I can use as a guide on how to add a List-paramter?